src/js/tabs/pid_tuning.js

```diff
@@ -547,8 +553,9 @@ TABS.pid_tuning.initialize = function (callback) {
             const dMinSwitch = $('#dMinSwitch');
             dMinSwitch.prop('checked', FC.ADVANCED_TUNING.dMinRoll > 0 || FC.ADVANCED_TUNING.dMinPitch > 0 || FC.ADVANCED_TUNING.dMinYaw > 0);
             dMinSwitch.change(function() {
-                const checked = $(this).is(':checked');
-                if (checked) {
+                const isdminchecked = $(this).is(':checked');
```

Comment:
indent here and close the }
```suggestion
                    const isDminChecked = $(this).is(':checked');
```